test/miscellaneous_tests/api.jl

```diff
@@ -100,6 +106,17 @@ let
     @test pmat == prodstoichmat(rnmat) == Matrix(prodstoichmat(rnmat, sparse = true))
 end
 
+# Tests `reactionparamsmap`, `reactionrates`, and `symmap_to_varmap` getters.
```

Comment:
Moved up from a different place. I haven't removed any tests nor written any actual new test blocks. However, for some files the order was rather jumbled. As part of the make I fixed this. Hopefully we won't have to deal with this again now.